### PR TITLE
Added data for 2017 edition, fixed refs #160 and #175

### PR DESCRIPTION
**Pull request by danielmocan**

### CODE REVIEW

File: app/data/2017/speakers.js [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+      description:
+        'Chunky is an opinionated framework that helps you write better React Apps by organizing functionality in modular, cross-functional chunks.',
+      message: '',
+      title: 'Introducing Chunky - The Full Stack React Framework',
+    },
+    twitter: 'https://twitter.com/idancali',
+    website: 'https://medium.com/@idancali',
+    workshop: null,
+  },
+  {
```
Comment (alexnm) [inline app/data/2017/speakers.js]: william didn't join in the end, you can remove him

File: app/data/2018/speakers.js [hunk 2]
```diff
@@ -297,7 +297,7 @@ export default [
     talk: {
       title: 'Advanced Redux - Design Patterns and Practices',
       description:
-        'Redux is as a modern popular state management library in frontend today. But if you dig a little deeper, you will find out the Redux build on top of patterns and concepts that was introduced 20 years ago. During this session, we talk about design patterns from the world of micro services and messaging systems and see how to use those ideas in the context of modern single page app. You will never use Redux the same way again.',
+        'CACACACa Redux is as a modern popular state management library in frontend today. But if you dig a little deeper, you will find out the Redux build on top of patterns and concepts that was introduced 20 years ago. During this session, we talk about design patterns from the world of micro services and messaging systems and see how to use those ideas in the context of modern single page app. You will never use Redux the same way again.',
```
Comment (alexnm) [inline app/data/2018/speakers.js]: This reminds me of someone :)) Please remove

File: app/data/menuitems.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ export default [
     links: [
       {
         id: 6.1,
-        url: 'http://2017.jsheroes.io/',
```
Comment (alexnm) [inline app/data/menuitems.js]: I would put /2018 first then /2017, to keep the reverse order on the long run

File: app/data/2018/speakers.js (hunk 2 above)
Comment (SaraVieira) [reply app/data/2018/speakers.js]: Leave it

Comment (danielmocan) [reply app/data/2018/speakers.js]: I don`t know who you are referring too :).
It was late, and I was receiving shity data, unwanted data :P, that was off-course hardcoded.

( Fixed )

File: app/data/menuitems.js (hunk 3 above)
Comment (danielmocan) [reply app/data/menuitems.js]: Done

File: app/data/2017/speakers.js (hunk 1 above)
Comment (danielmocan) [reply app/data/2017/speakers.js]: done

**Pull request merged**